etc/schema.json

```diff
@@ -5184,8 +5190,29 @@
                 "defrag": {
                     "type": "object",
                     "properties": {
-                        "max_frag_hits": {
-                            "type": "integer"
+                        "tracker_soft_reuse": {
+                            "type": "integer",
+                            "description":
+                                    "How many times a tracker was force reused after it was complete"
+                        },
+                        "tracker_hard_reuse": {
+                            "type": "integer",
+                            "description":
+                                    "How many times a tracker was force reused before it was done"
+                        },
```

Comment:
I'd like further explanation of what these really mean? If only in the commit.

Reply:
Open to better naming.

"tracker_soft_reuse" spare pool was empty and memcap is reached, so the packet thread that needed a tracker went into hash directly to reuse a tracker from the hash, and it found one that was already marked `remove` (so reassembly finished).

"tracker_hard_reuse" spare pool was empty and the memcap is reached, so the packet thread that needed a tracker went into hash directly to reuse a tracker from the hash, but it found one that was still in use (`remove` false) and reused that. The original tracker is cleared leading to the packet for that tracker not being reassembled.

Reply:
Maybe description updates:

### tracker_soft_reuse

Finished tracker re-used from hash table before being moved to spare pool.

### tracker_hard_reuse

Active tracker force closed before completion and reused for new tracker.

Reply:
see https://github.com/OISF/suricata/pull/11224